cg/clients/freshdesk/freshdesk_client.py

```diff
@@ -38,6 +38,7 @@ def create_ticket(self, ticket: TicketCreate) -> TicketResponse:
             return TicketResponse.model_validate(response.json())
         except RequestException as error:
             LOG.error(f"Could not create ticket: {error}")
+            LOG.error(f"Response: {response.text}")
```

Comment:
This error handling and logging should be extracted to a separate module

Reply:
👍